transaction-ballerina/tests/transactional_expr.bal

```diff
@@ -17,6 +17,7 @@
 import ballerina/test;
 
 @test:Config {
```

Comment:
Need to enable after fixing https://github.com/ballerina-platform/ballerina-lang/issues/43608